spec/output/generator_spec_go/test.x/MyXDR_generated.go

```diff
@@ -547,7 +547,7 @@ var _ xdrType = (*Hashes2)(nil)
 //
 type Hashes3 []Hash
 // EncodeTo encodes this value using the Encoder.
-func (s *Hashes3) EncodeTo(e *xdr.Encoder) error {
+func (s Hashes3) EncodeTo(e *xdr.Encoder) error {
   var err error
 if _, err = e.EncodeUint(uint32(len(s))); err != nil {
```

Comment:
I think we should try to compile the generated files. This doesn't compile.

Reply:
+1

Reply:
💯  It would be really helpful if we tested the outputs. We should probably do so for the go and rust generated code at least.